elixir/apps/api/lib/api/client/views/resource.ex

```diff
@@ -14,7 +15,9 @@ defmodule API.Client.Views.Resource do
       id: resource.id,
       type: :cidr,
       address: address,
-      name: resource.name
+      address_description: resource.address_description,
+      name: resource.name,
+      gateway_groups: Views.GatewayGroup.render_many(resource.gateway_groups)
```

Comment:
@conectado resource now will also return `address_description` and `gateway_groups`, do we need to do anything to make sure it won't crash on de-serialization?

Reply:
ping @conectado

Reply:
Sorry, I missed this.

I'm surprised that CI didn't fail

Reply:
Ah, my bad, I though by default serde denied unknown fields, but it's the other way around.

We're fine like this